src/Processors/Formats/Impl/ParallelFormattingOutputFormat.h

```diff
@@ -74,18 +74,23 @@ class ParallelFormattingOutputFormat : public IOutputFormat
         , pool(params.max_threads_for_parallel_formatting)
 
     {
+        LOG_TRACE(&Poco::Logger::get("ParallelFormattingOutputFormat"), "Parallel formatting is being used");
```

Comment:
Maybe `LOG_TEST`?

Reply:
I'd better to remove this line, because parallel formatting is used almost always after changes from @Avogar